internal/studio/studioHandlers.go

```diff
@@ -402,10 +395,6 @@ func (h *StudioHandlers) exit(ctx context.Context, w http.ResponseWriter, r *htt
 	return nil
 }
 
-// ---------------------------------
-// Helper functions
-// ---------------------------------
-
 func (h *StudioHandlers) convertLastRunResult(ctx context.Context, step string) (*components.RunResponseData, error) {
```

Comment:
why not move this one out? Maybe because it can't? Ideally every method / function in this file is a handler to a route